ketchupbot-discord/GalaxyGPT.cs

```diff
@@ -153,7 +155,7 @@ await message.ReplyAsync("""
                 // NOTE: These numbers are hardcoded and not necessarily representative of the actual costs, as the model can change
                 if (questionTokens != 0 && responseTokens != 0)
                     answerMessage.AppendLine(
-                        $"Cost: ${Math.Round(questionTokens * 0.00000015 + responseTokens * 0.0000006, 10)}");
```

Comment:
I don't really see why we remove the dollar sign. Does this somehow get rid of the scientific notation?

Reply:
I made it round to like 6 places or something and i put it in terms of cents, not dollars so theres less 0s. also multiplied every constant there by 100 to keep the value the same

Reply:
oh nice. can you get rid of the "cents" and use ¢

Reply:
I was considering that, but idk if c# likes the cent symbol in strings or if I have to do some u/ bs so I went safe lol

Reply:
nah you'll be fine. c# string interpolation works like python's rather than js's